src/lib/chimp-spec.js

```diff
@@ -0,0 +1,75 @@
+const Chimp = require('./chimp');
```

Comment:
I'm moving all the spec files to be right next to their source files so your assumption is spot on. Thanks :)